ansible/roles/debops.discourse/nginx/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+# FIXME: What is the proper way to add this file?
+# FIXME: Are these the correct tags? Is this the correct way to use tags?
```

Comment:
I think that this can stay for now, until `debops.nginx` role will have proper support for snippets.